plugin/src/main/java/org/opensearch/ml/model/MLModelManager.java

```diff
@@ -1281,20 +1281,44 @@ private synchronized void deployControllerWithDeployingModel(
                 // well
                 listener
                     .onResponse(
-                        "The model "
+                        "No controller is deployed because the model "
                             + modelId
-                            + " is expected not having a model controller. Please use the create model controller api to create one if this is unexpected."
+                            + " is expected not having a model controller. Please use the create controller api to create one if this is unexpected."
```

Comment:
> is expected not having a model controller

This is conflicting with your checking `else if (mlModel.getIsControllerEnabled() == null || !mlModel.getIsControllerEnabled())` 

Your checking means `model doesn't have model controller enabled `

`mlModel.getIsControllerEnabled() == null || !mlModel.getIsControllerEnabled()` can be replaced with `BooleanUtils.isFalse(mlModel.getIsControllerEnabled())`

Reply:
Changed message.